### PR TITLE
Add schedule section to the tutorial

### PR DESCRIPTION
**Pull request by codewithcheese**
Hi, when trying to understand scheduling I referred to the tutorial. I think a docs section on putting airflow into production would be great. But, for now a quick mention of 'airflow schedule' seems appropriate in the tutorial.

### CODE REVIEW

File: docs/tutorial.rst [hunk 1]
```diff
@@ -370,6 +370,15 @@ start_date specified itself, for which this dependency is disregarded.
     # start your backfill on a date range
     airflow backfill tutorial -s 2015-01-01 -e 2015-01-07
 
+Schedule
+''''''''
+``airflow schedule`` needs to be running for the tasks in your DAGs to be
```
Comment (mistercrunch) [inline docs/tutorial.rst]: Scheduler
``````````````
``airflow scheduler``

**Comment by mistercrunch** [comment]
Great, you could also link to this section:
http://pythonhosted.org/airflow/scheduler.html

Lookup how links work in rst. I'm still on the fence on whether I like rst or not...

**Comment by codewithcheese** [comment]
Updated. I use the doc reference linking style. Inspired by the tutorial link in start.rst

**Comment by mistercrunch** [comment]
`s/schedule /scheduler /g`

**Comment by codewithcheese** [comment]
Ahh, oops fixed.

**Comment by mistercrunch** [comment]
![img] (http://media.giphy.com/media/FqT0hV14vYnq8/giphy.gif)

**Pull request merged**